eng/pipelines/templates/steps/analyze.yml

```diff
@@ -3,15 +3,18 @@ parameters:
   ServiceDirectory: ''
   TestMarkArgument: ''
   AdditionalTestArgs: ''
+  Artifacts: []
 
 steps:
   - template: /eng/pipelines/templates/steps/analyze_dependency.yml
 
-  - task: PythonScript@0
-    displayName: 'Verify Change Log'
-    inputs:
-     scriptPath: 'scripts/devops_tasks/verify_change_log.py'
```

Comment:
Is it time to also remove the verify_change_log.py script?

Reply:
@praveenkuttappan